shared/Mocks/StartupOpenIdConnectTesting.cs

```diff
@@ -42,8 +42,10 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.Configure<AppSettings>(Configuration.GetSection("AppSettings"));
 
-            //Sql client not available on mono
-            var useInMemoryStore = _platform.IsRunningOnMono;
+            //Sql client not available on mono, non-windows, or nano
+            var useInMemoryStore = !_platform.IsRunningOnWindows
```

Comment:
Can we just copy this property into `Platform` instead of repeating it multiple times over?